### PR TITLE
uppgrade to ng5 bs4

### PR DESCRIPTION
**Pull request by einariBOF**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -113,25 +105,15 @@ AND  I expect the data to be saved
...
-
-    $> npm run-script serve
+    $> npm start
+```
 
-4. Visit the page in browser
+Visit the page in http://localhost:4200
+Profile API can be reached at http://localhost:3000
 
-    http://localhost:3000
```
Comment (ghost) [inline README.md:119]: Under `Instructions & Deliverables` there's a sentence `You may use any google or other references for angular/bootstrap syntax`. 

I think we should change this to `You may use Google or any other references for angular/bootstrap syntax`

Comment (ghost) [reply README.md:119]: Sorry one more,  in **Other Notes**

I think this reads better: `Solution should be fully working when we check out` **your**` code`

**Comment by ghost** [comment]
It works 👍

**Comment by spinx** [comment]
@einariBOF can we also commit package-lock.json to make sure the dependencies work in the future for sure as well

**Review by spinx — APPROVED** [review]

**Pull request merged**